arrows.c

```diff
@@ -0,0 +1,51 @@
+#include "arrows.h"
```

Comment:
No możesz zrobić jeden plik .h z tymi obrazkami, up to you. Tylko wtedy te tablice do jednego pliku wrzucasz